scripts/testone.sh

```diff
@@ -0,0 +1,27 @@
+set -ex
```

Comment:
Can you add a #!/bin/bash and run ShellCheck on this script to catch other lints?